.storybook/main.js

```diff
@@ -9,7 +9,8 @@ module.exports = {
     "@storybook/addon-links",
```

Comment:
the last commit says `MI-2220: Changes in package lock` this is misleading as it also contains review fixes, try putting more relevant messages like
`[MI-2220]: Changes in package-lock and review fixes`